### PR TITLE
Remove scripted opex template

### PR DESCRIPTION
**Pull request by ImplicitDeny**
Also remove the old scripted GZM, due to it being replaced by the modernized, modded version https://github.com/team-gsri/GZM

### CODE REVIEW

File: description.ext [hunk 1]
```diff
@@ -81,63 +45,6 @@ class CfgFunctions {
...
-	class HeliMoved : HeliInfo
-	{
-		description = $STR_GSRI_heliMoved;
-	};
-	class HeliFRIES : HeliInfo
-	{
-		description = $STR_GSRI_heliFRIES;
-	};
-
 	// Todo clean tout ça
```
Comment (ArwynFr) [inline description.ext:48]: Is there an open issue for this TODO ?

File: init.sqf [hunk 2]
```diff
@@ -1,14 +1,5 @@
-// Reminder : add D-651 Normandie tag french flag with Eden properties
-
-// Tracking Zeus commands
-["zeus", {(_this select 0) spawn GSRI_fnc_handleRequest}, "all"] call CBA_fnc_registerChatCommand;
-
 //EOS SYSTEM
```
Comment (ArwynFr) [inline init.sqf:1]:
```suggestion
//ENEMY OCCUPATION SYSTEM
```

File: description.ext (hunk 1 above)
Comment (ImplicitDeny) [reply description.ext:48]: Now created #25

**Comment by ImplicitDeny** [comment]
Also, will fix #24 when merged.

**Pull request merged**